src/tree/SubscriptionTreeItem.ts

```diff
@@ -109,8 +109,10 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         });
 
         const gotRemote: boolean = !!wizardContext.repoHtmlUrl;
+        wizardContext.telemetry.properties.gotRemote = String(gotRemote);
+
         wizardContext.fsPath = wizardContext.fsPath || getSingleRootFsPath();
-        addWorkspaceTelemetry(wizardContext);
```

Comment:
Some of the telemetry wasn't relevant anymore, specifically isGitProject and isGitInstalled since we validate that upfront now.  Since we were just keeping track of if we got a remote and the num of workspaces, I just moved that into the `SubscriptionTreeItem`.